examples/games/tetris.code

```diff
@@ -0,0 +1,493 @@
+// -- Controls:
+// Confirm input by pressing enter. (Input is line buffered)
+// Moving. a = left. s = down. d = right
+// Use uppercase A or D to move left or right 2 steps. For down S and s both work.
+// Every move makes the piece go down by 1.
+// To only move down 1 step. Just hit enter.
+// Rotation: Q or q for Counter-Clockwise. E or e for Clockwise
+
+// -- Differences:
+// Board height is 12 instead 20 since its a bit slow to draw and it's tall in the terminal.
+// You have as much time as you want between moves.
+// You can max move 2 steps in every direction except up.
+
+// -- Tricks:
+// Uses (cond1) + (cond2) instead of (cond1 || cond2). This is faster
+// For Loops uses (i - num) instead of (i < num). Same as (i != num). Much faster and smaller.
+// Uses (var - num) instead of (var != num). Smaller and faster
+// Nested IFs are smaller and faster than && since there is no short-circuit boolean evaluation
+// Uses (var%%) to clear while using the data
```

Comment:
I created the %% operator as a joke (modulo 1 is always 0, lol)
I didn't think there could be a legitimate use for it, but you found a way, and I'm very pleased and impressed :)